ci-operator/step-registry/baremetalds/packet/setup/baremetalds-packet-setup-commands.sh

```diff
@@ -92,11 +93,13 @@ cat > packet-config.yaml <<-EOF
         done
       dest: "${SHARED_DIR}/packet-conf.sh"
 EOF
+latest_pip=$(find /usr/bin/ -name "pip*" -type f  | grep -oE '[0-9]+\.[0-9]+' | sort -V | tail -n1)
+pip"${latest_pip}" install -r https://raw.githubusercontent.com/equinix-labs/ansible-collection-equinix/main/requirements.txt
+ansible-galaxy collection install equinix.cloud
```

Comment:
can we move the collection installation to the image running this script?

Reply:
ya, looks like we end up installing it twice if doing it here

Reply:
Done https://github.com/openshift-metal3/dev-scripts/pull/1633